documentation/modules/exploit/multi/php/car_rental_system.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+Car Rental System 1.0 is a Web Application to rent cars. It was write using PHP and MySQL.
+
+Vendor Homepage: https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html
+Software Link: https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code
+
+to install, extract all files in a web directory, edit admin/db_connect.php, and create database following database/car_rental_db.sql
```

Comment:
```suggestion
To install Car Rental System 1.0, extract all files from the software link above into a web directory, edit admin/db_connect.php, and create database following database/car_rental_db.sql
```
Made a slight suggestion above but these directions aren't clear enough. What changes need to be made to `admin/db_connect.php`? Also if you have directions for creating the database you should list them here.